src/main/java/net/greghaines/jesque/worker/Watchdog.java

```diff
@@ -124,6 +143,17 @@ private void activateIsAliveService(final String serverName) throws RuntimeExcep
         scheduler.scheduleAtFixedRate(isAlive, INITIAL_DELAY, IS_ALIVE_PERIOD, SECONDS);
     }
 
+    private void heartbeatIsAlive(final Jedis jedis) {
+
+        final String recoveryStatus = redisRestartRecoveryOn ? RecoveryStatus.HEARTBEAT.name() : RecoveryStatus.DISABLED.name();
```

Comment:
We aren't sending here the recovery-status but we're sending is the recovery mechanism is enabled or not.
Let's rename accordingly and avoid using the enum which is related to the status.